sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/javaCode.scala

```diff
@@ -113,6 +113,21 @@ object JavaCode {
   def isNullExpression(code: String): SimpleExprValue = {
     expression(code, BooleanType)
   }
+
+  /**
+   * Create an `InlineBlock` for Java Class name.
+   */
+  def className(javaClass: Class[_]): InlineBlock = InlineBlock(javaClass.getName)
```

Comment:
I am not sure about the names of these 3 methods... They are doing kind of the same thing but this name is pretty different form the other 2... What do you think?

Reply:
Is `javaClass` better for you?

Reply:
what about `javaType` as the method below? They return the same thing, just with a different input, don't they?

Reply:
Ok. Looks good to me.